ffmpeg/ffmpeg_test.go

```diff
@@ -1869,3 +1869,45 @@ func TestResolution_Clamp(t *testing.T) {
 	checkError = require.Error
 	test(l, Size{300, 600}, portrait, Size{300, 600})
 }
+
+func TestDurationFPS_GetCodecInfo(t *testing.T) {
+	run, dir := setupTest(t)
+	defer os.RemoveAll(dir)
+
+	//Generate test files
+	cmd := `
+	cp "$1/../data/bunny.mp4" test.mp4
+	cp "$1/../data/duplicate-audio-dts.ts" test.ts
+	ffmpeg -loglevel warning -i test.mp4 -vn -c:a flac test.flac
```

Comment:
This is good, it would be nice to also add checks for expected values from ffprobe - helps ensure that our own calculations are correct

also just checking, which one of these triggers the no-duration case that requires manual copying? (that *should* also surface via ffprobe, eg a duration of "N/A")

Reply:
It was with webm files. `ffprobe -show_streams test.webm | grep 'duration='` gives **N/A** for duration. I've added webm to the test, it is working with the latest changes.

Reply:
Added the ffprobe commands, shortened all the videos to 2 seconds. The int64 change is working well so I removed `calculate_stream_duration`. I am happy with rounding numbers for now

Reply:
>  I am happy with rounding numbers for now

Double checking (no pun intended) ... anything less than 1 (eg, 0.99) will be rounded down to 0, 1.99 becomes 1, etc. That seems pretty significant (eg, how do you distinguish between "very short" and "zero duration") but if is still OK for your purposes, then 👍 

Otherwise, I believe we can still match the values [calculated by ffprobe](https://github.com/FFmpeg/FFmpeg/blob/master/fftools/ffprobe.c#L952) - our equivalent would be `out->dur = ic->duration * av_q2d(AV_TIME_BASE_Q)` ... mathematically the same as the old code, but I don't know if the computed precision would differ (and it would only come into play for very large durations, anyhow)